libmscore/note.h

```diff
@@ -53,6 +53,21 @@ static const int MAX_DOTS = 4;
 class NoteHead final : public Symbol {
       Q_GADGET
    public:
+      enum class Scheme : signed char {
```

Comment:
Need to add a comment that this enum should be synchronized with `static NoteHeadName noteHeadSchemeNames[]`

Reply:
[Group](https://github.com/mattmcclinch/MuseScore/blob/bd64822d8fb3e7f18dd67fc4664dd6bd4e6335b3/libmscore/note.h#L71) does not have such a comment. Neither does [Type](https://github.com/mattmcclinch/MuseScore/blob/bd64822d8fb3e7f18dd67fc4664dd6bd4e6335b3/libmscore/note.h#L144).

Reply:
And it's very bad ;)

Reply:
Could you please add comments here and in other similar places that you looked.

Reply:
Done. I also changed `QString s` to `const QString& s` in the function declarations/definitions for `name2scheme`, `name2group`, and `name2type`, and I changed the `Q_ASSERT` in `setHeadScheme` to `IF_ASSERT_FAILED`.